kubernetes/client/clientset/typed/compose/v1beta1/stack.go

```diff
@@ -46,44 +45,44 @@ func newStacks(c *ComposeV1beta1Client, namespace string) *stacks {
 }
 
 // Create takes the representation of a stack and creates it.  Returns the server's representation of the stack, and an error, if there is any.
-func (c *stacks) Create(stack *v1beta1.Stack) (result *v1beta1.Stack, err error) {
-	result = &v1beta1.Stack{}
-	err = c.client.Post().
+func (c *stacks) Create(stack *v1beta1.Stack) (*v1beta1.Stack, error) {
+	result := &v1beta1.Stack{}
+	err := c.client.Post().
 		Namespace(c.ns).
 		Resource("stacks").
 		Body(stack).
 		Do().
 		Into(result)
-	return
+	return result, err
 }
 
 // Update takes the representation of a stack and updates it. Returns the server's representation of the stack, and an error, if there is any.
-func (c *stacks) Update(stack *v1beta1.Stack) (result *v1beta1.Stack, err error) {
-	result = &v1beta1.Stack{}
-	err = c.client.Put().
+func (c *stacks) Update(stack *v1beta1.Stack) (*v1beta1.Stack, error) {
+	result := &v1beta1.Stack{}
+	err := c.client.Put().
 		Namespace(c.ns).
 		Resource("stacks").
 		Name(stack.Name).
 		Body(stack).
 		Do().
 		Into(result)
-	return
+	return result, err
 }
 
 // UpdateStatus was generated because the type contains a Status member.
 // Add a +genclientstatus=false comment above the type to avoid generating UpdateStatus().
```

Comment:
Looks like `UpdateStatus()` isn't used? should we remove this? (same for the other API version)

Reply:
Same answer 😄